### DISCUSSION TITLE
Add a changelog.md

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmcv#2343] [General]
文档推荐在 readthedocs 阅读。没有 changelog.md，  readthedocs 里就找不到更新日志了。加一下感觉会好一点？

The documentation is recommended for reading at readthedocs. Without changelog.md, you can't find the update log in readthedocs. Would it feel better to add it?

**Comment by RangeKing** [comment]
I think it is better to unify the documentation of each repo, especially in OpenMMLab 2.x.
Reference Link: https://mmdetection.readthedocs.io/en/latest/changelog.html
